plugins/snapshot.yaml

```diff
@@ -0,0 +1,109 @@
+      to:  "."
+    - from: "kubectl-snapshot-*/LICENSE"
+      to: "."
+    bin: "./kubectl-snapshot-linux-amd64"
+  - selector:
+      matchLabels:
+        os: linux
+        arch: 386
+    uri: https://github.com/fbrubbo/kubectl-snapshot/archive/v0.1.2.tar.gz
+    sha256: "3a9cb891eb3d0b20aa42d94c139ac27c6b44ecf9d42c8e844d3ea6c7dc267d20"
+    files:
+    - from: "kubectl-snapshot-*/bin/kubectl-snapshot-linux-386"
+      to:  "."
+    - from: "kubectl-snapshot-*/LICENSE"
+      to: "."      
+    bin: "./kubectl-snapshot-linux-386"
+
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/fbrubbo/kubectl-snapshot/archive/v0.1.2.tar.gz
+    sha256: "3a9cb891eb3d0b20aa42d94c139ac27c6b44ecf9d42c8e844d3ea6c7dc267d20"
+    files:
+    - from: "kubectl-snapshot-*/bin/kubectl-snapshot-darwin-amd64"
+      to:  "."
+    - from: "kubectl-snapshot-*/LICENSE"
+      to: "."
+    bin: "./kubectl-snapshot-darwin-amd64"
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: 386
+    uri: https://github.com/fbrubbo/kubectl-snapshot/archive/v0.1.2.tar.gz
+    sha256: "3a9cb891eb3d0b20aa42d94c139ac27c6b44ecf9d42c8e844d3ea6c7dc267d20"
+    files:
+    - from: "kubectl-snapshot-*/bin/kubectl-snapshot-darwin-386"
+      to:  "."
+    - from: "kubectl-snapshot-*/LICENSE"
+      to: "."
+    bin: "./kubectl-snapshot-darwin-386"
+    
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/fbrubbo/kubectl-snapshot/archive/v0.1.2.tar.gz
+    sha256: "3a9cb891eb3d0b20aa42d94c139ac27c6b44ecf9d42c8e844d3ea6c7dc267d20"
+    files:
+    - from: "kubectl-snapshot-*/bin/kubectl-snapshot-windows-amd64.exe"
+      to:  "."
+    - from: "kubectl-snapshot-*/LICENSE"
+      to: "."
+    bin: "./kubectl-snapshot-windows-amd64.exe"
+  - selector:
+      matchLabels:
+        os: windows
+        arch: 386
+    uri: https://github.com/fbrubbo/kubectl-snapshot/archive/v0.1.2.tar.gz
+    sha256: "3a9cb891eb3d0b20aa42d94c139ac27c6b44ecf9d42c8e844d3ea6c7dc267d20"
+    files:
+    - from: "kubectl-snapshot-*/bin/kubectl-snapshot-windows-386.exe"
+      to:  "."
+    - from: "kubectl-snapshot-*/LICENSE"
+      to: "."
+    bin: "./kubectl-snapshot-windows-386.exe"    
+
+  shortDescription: >-    
+    Prints a snapshot of nodes, pods and HPAs resource usage
+  homepage: https://github.com/fbrubbo/kubectl-snapshot
+  description: |
+    This plugin takes a snapshot of resources (ie. cpu and memory) usage for 
+    pods, HPAs, deployments without HPA and nodes. Other best configuration 
+    practices, such as Pod Disruption Budget, probes and etc are also included. 
+    This is specialy useful to understand, at scale, what you have configured 
+    in your cluster. 
+
+    Usage:
+      kubectl snapshot
+      kubectl snapshot -csv-output <NAME>
+
+    Other fine tunning are allowed, to understand more type:
+      kubectl snapshot -h
+
+  caveats: |
```

Comment:
IMHO, this information is still better suited for general documentation and not the caveats section. The caveats section is really meant for pre-requisites or necessary rights to run this plugin. For example, you should mention that it requires a metrics collection component (heapster) in the cluster.

Reply:
Agreed. But as I mention to ahmetb, the caveat section is the only one being printed when a user installs the plugin. To be honest, I couldn't figure out where the description field is being used. 
Anyways, I will follow your guidance and add everything in the description section.